pkg/services/subscriptions/models.go

```diff
@@ -1,50 +1,41 @@
-/*
-The API response for /subscriptions look as follows (p.s. the uuids are fake, so good luck authenticating HACKERS).
-If you'd like to make your own API requests, you can follow this guide to get you started:
-https://medium.com/@mauridb/calling-azure-rest-api-via-curl-eb10a06127
-
-GET:
-➜ curl -s -X GET -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" "https://management.azure.com/subscriptions?api-version=2020-01-01" | jq .
-{
-  "value": [
-    {
-      "id": "/subscriptions/b5549535-3215-4868-a289-f80095c9e718",
-      "authorizationSource": "RoleBased",
-      "managedByTenants": [],
-      "subscriptionId": "b5549535-3215-4868-a289-f80095c9e718",
-      "tenantId": "b5549535-3215-4868-a289-f80095c9e718",
-      "displayName": "Pay-As-You-Go",
-      "state": "Enabled",
-      "subscriptionPolicies": {
-        "locationPlacementId": "Public_2014-09-01",
-        "quotaId": "PayAsYouGo_2014-09-01",
-        "spendingLimit": "Off"
-      }
-    }
-  ],
-  "count": {
-    "type": "Total",
-    "value": 1
-  }
-}
-*/
-
 package subscriptions
 
+import (
+	"fmt"
+)
+
 // SubscriptionResponse models the subscription response from the API
 type SubscriptionResponse struct {
 	Value []Subscription `json:"value"`
 }
 
 // Subscription is the object we store in our Inventory grab bag to model a subscription
 type Subscription struct {
-	ID                   string `json:"id"`
-	SubscriptionID       string `json:"subscriptionId"`
-	DisplayName          string `json:"displayName"`
-	State                string `json:"state"`
-	SubscriptionPolicies struct {
-		LocationPlacementID string `json:"locationPlacementId"`
-		QuotaID             string `json:"quotaId"`
-		SpendingLimit       string `json:"spendingLimit"`
-	} `json:"subscriptionPolicies"`
+	ID                   string               `json:"id"`
+	SubscriptionID       string               `json:"subscriptionId"`
+	DisplayName          string               `json:"displayName"`
+	State                string               `json:"state"`
+	SubscriptionPolicies SubscriptionPolicies `json:"subscriptionPolicies"`
+}
+
+// SubscriptionPolicies is a model for subscriptions, and allows us to easily modify Subscription responses
+type SubscriptionPolicies struct {
+	LocationPlacementID string `json:"locationPlacementId"`
+	QuotaID             string `json:"quotaId"`
+	SpendingLimit       string `json:"spendingLimit"`
+}
+
+// NewSubscription takes a string ID and returns a basic Subscription object
+func NewSubscription(subscriptionID string) Subscription {
```

Comment:
Related to my comment above about having raw JSON in a dir/package, would it be easier to do that and return an unmarshalled JSON here instead of manually coding the JSON structure in the constructors? We could override any desired properties after the unmarshall prior to the `return` to allow for custom things like `subscriptionID` as you've done here.